### ISSUE TITLE
PropTypes Warnings on expo-web

### ISSUE THREAD

**Issue by jaxoncreed** [calintamas/react-native-toast-message#117]
When you run this library in Expo Web, you get warnings for every time `ViewPropTypes.style` is used. For example:

```
Warning: Failed prop type: Toast: prop type `style` is invalid; it must be a function, usually from the `prop-types` package, but received `object`.
    in Toast (at App.tsx:40)
    in App (created by ExpoRootComponent)
    in ExpoRootComponent (created by RootComponent)
    in RootComponent
    in div (created by View)
    in View (created by AppContainer)
    in div (created by View)
    in View (created by AppContainer)
    in AppContainer
```

Happens because of the PropTypes here https://github.com/calintamas/react-native-toast-message/blob/master/src/index.js#L383

This is probably because prop-types has been depricated on React-Native-Web, but not React-Native. https://github.com/necolas/react-native-web/issues/1537#issuecomment-601167089

**Comment by jaxoncreed** [comment]
I made a temporary fix here https://github.com/jaxoncreed/react-native-toast-message, but all it does is remove the offending PropTypes, so it's by no means a full fix.

**Comment by arealmaas** [comment]
Yes, similar to this issue I guess: https://github.com/GeekyAnts/NativeBase/issues/3264

**Comment by calintamas** [comment]
oh, I was not aware of this deprecation. I'll take a look into it soon. thanks for raising the issue!

**Comment by seanadkinson** [comment]
Yeah, looks like we'll need the prop-types stuff dropped in order to support the latest `react-native-web`.  In the meantime, following the example in the linked `react-native-web` issue, here is a script that will monkey-patch the library:

**fix-rn-web-proptypes.sh**:
```bash
#!/bin/bash
# Only need this until this issue is fixed: https://github.com/calintamas/react-native-toast-message/issues/117

echo 'Fixing PropTypes issues'

if grep -q "export const ViewPropTypes = { style: null };" ./node_modules/react-native-web/dist/index.js; then
    echo "ViewPropTypes fixed already!"
else
    printf "\nexport const ViewPropTypes = { style: () => {} };">> ./node_modules/react-native-web/dist/index.js
    printf "\nexport const ViewPropTypes = { style: () => {} };">> ./node_modules/react-native-web/src/index.js
    printf "\nText.propTypes = { style: () => {} };">> ./node_modules/react-native-web/dist/exports/Text/index.js
    printf "\nText.propTypes = { style: () => {} };">> ./node_modules/react-native-web/src/exports/Text/index.js
fi
```

Then add `"postinstall": "./fix-rn-web-proptypes.sh"` in the `scripts` section of your `package.json`.

After doing the above, I'm able to use this package with `react-native-web` 👍

**Comment by calintamas** [comment]
published a fix in v1.4.5
